torrentool/torrent2.py

```diff
@@ -0,0 +1,444 @@
+from os.path import join, isdir, getsize, normpath, basename, getsize
```

Comment:
A new file - hard to see was have changed, and what is new.
Classic approach is to make several braches (based on one another is they are connected) with gradual improvements introduction.